python/pyspark/pipelines/spark_connect_graph_element_registry.py

```diff
@@ -110,3 +115,10 @@ def register_sql(self, sql_text: str, file_path: Path) -> None:
         command = pb2.Command()
         command.pipeline_command.define_sql_graph_elements.CopyFrom(inner_command)
         self._client.execute_command(command)
+
+def source_code_location_to_proto(
+    source_code_location: SourceCodeLocation,
+) -> pb2.SourceCodeLocation:
+    return pb2.SourceCodeLocation(
+        file_name=source_code_location.filename, line_number=source_code_location.line_number
+    )
```

Comment:
I think running `dev/reformat-python` will add a newline at the end of this file.